lib/stages/HFBAccumulate.hpp

```diff
@@ -27,6 +30,8 @@
  * Note: _num_frames_to_integrate cannot go below 16 frames as _num_frames_to_integrate cannot be
  * lower than the max_frames_missing
  *
+ * This stage will also calculate the within sample variance for weights.
+ *
  * The output of this stage is written to a raw file where each chunk is
  * the metadata followed by the frame and indexed by frequency ID.
  * This raw file is then transposed and compressed into a structured
```

Comment:
The docstring here doesn't list the cls buffer as an input.

Reply:
Thanks Richard! :+1: